### DISCUSSION TITLE
Microsoft Defender SmartScreen preventing OrcaSlicer?

### DISCUSSION THREAD

**Discussion by SimonThompson1965** [OrcaSlicer/OrcaSlicer#2831] [Q&A]
I just tried to install OrcaSlicer for the first time, but Microsoft Defender keeps blocking it when I run the installer, or when I try the portable zip file, giving me the following error:

Windows protected your PC
Microsoft Defender SmartScreen prevented an unrecognised app from starting. Running this app might put your PC at risk.

Application: 
OrcaSlicer_Windows_Installer_1.8.0.exe 
Publisher:  
Unknown publisher

I'm assuming it's actually safe to run, but would be good to know for sure before I do. For info I'm currently running Bambu Studio 01.07.07.89 which installed with no issues, I'm using Windows 10 Pro 22H2 64bit and have Defender updated with latest definitions.

**Comment by Sortam** [comment]
![image](https://github.com/SoftFever/OrcaSlicer/assets/45393361/da408f07-94bd-41cf-82b0-ed74bbd25802)
If you compare the properties of the installer files for both applications, you'll see one difference - Bambu Studio has the "Digital Signatures" tab, which is missing in OrcaSlicer. This means that the OrcaSlicer installer is not signed by the developer's certificate and its integrity can't be verified. That's why Windefender warns you - it can't verify the source of this file and whether someone has tampered with it before you get it. 
Implementing such a signature for OrcaSlicer will cost money and effort and is really not necessary. As long as you downloaded the installer from the official Github release page you are safe and the warning can be ignored. 
I hope this cleared things up a bit.

**Comment by AdamTV** [comment]
![image](https://github.com/user-attachments/assets/70daf40f-ce2b-4cd4-adb6-7eb2e34aded0)

**Reply by Sawtaytoes** [reply]
Is it really this easy? If so, why haven't Orca done it? The URL is blocked for me via bad lists, so I don't wanna touch it.
